frontend/javascripts/admin/dataset/dataset_add_view.tsx

```diff
@@ -317,7 +317,6 @@ const getPostUploadModal = (
                 <Button type="primary" onClick={() => history.push("/jobs")}>
                   View the Jobs Queue
                 </Button>
-                import
```

Comment:
what’s this?

Reply:
this was accidentally added in https://github.com/scalableminds/webknossos/commit/a58a78ee5ddcf31b4cc06c850c8a8f082104b222#diff-b626f1738c0fb701d260efe19bdee10f7f57981f881c85a42d3588999c27db55R320 and I removed it again.